conductor.json

```diff
@@ -90,7 +90,23 @@
         "SQS_REGION": "{{env.SQS_REGION}}",
         "SQS_ACCESS_KEY": "{{env.SQS_ACCESS_KEY}}",
         "SQS_SECRET": "{{env.SQS_SECRET}}",
-        "SQS_URL": "http://{{services.sqs.host}}:{{services.sqs.port}}/queue/asl-dev"
+        "SQS_URL": "http://{{services.sqs.host}}:{{services.sqs.port}}/queue/asl-dev",
+        "KEYCLOAK_REALM": "{{env.KEYCLOAK_REALM}}",
+        "KEYCLOAK_URL": "{{env.KEYCLOAK_URL}}",
+        "KEYCLOAK_CLIENT": "{{env.KEYCLOAK_CLIENT}}",
+        "KEYCLOAK_SECRET": "{{env.KEYCLOAK_SECRET}}",
+        "EMAILER_SERVICE": "http://{{services.asl-emailer.host}}:{{services.asl-emailer.port}}",
+        "JWT_SECRET": "{{env.JWT_SECRET}}"
```

Comment:
I think resolver also needs a username and password for logging into keycloak to perform admin actions.